packages/manager/apps/dedicated/client/app/account/user/users/sso/delete/user-users-sso-delete.controller.js

```diff
@@ -0,0 +1,42 @@
+import get from 'lodash/get';
+
+export default class UserAccountUsersSSODeleteCtrl {
+  /* @ngInject */
+  constructor($scope, UseraccountUsersService, Alerter, $translate) {
```

Comment:
try to group deps by nature:
```suggestion
  constructor($scope, $translate, Alerter, UseraccountUsersService) {
     this.$scope = $scope;
     this.$translate = $translate;
     this.alerter = Alerter;
     this.usersService = UseraccountUsersService;
```

Reply:
Done